prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-expressions */
+const number1 = 1;
+const number2 = 2;
+number1 * number2;
+const product = 2;
+console.log('value of product:', product);
+console.log('type of product:', typeof product);
+const charge = 10;
+const payment = 6;
+charge - payment;
+const amountRemaining = 4;
+console.log('value of amountRemaining:', amountRemaining);
+console.log('type of amountRemaining:', typeof amountRemaining);
+const tests = 12;
+const assignments = 15;
+const final = 18;
+(tests + assignments + final) / 3;
+const grade = 15;
+console.log('value of grade:', grade);
+console.log('type of grade:', typeof grade);
+const firstName = 'Stefanny';
+const lastName = 'Loversky';
+`${firstName} ${lastName}`;
+const fullname = 'Stefanny Loversky';
+console.log('value of fullname:', fullname);
+console.log('type of fullname:', typeof fullname);
+const pH = 8;
+pH > 7;
+const isAcidic = true;
+console.log('value of isAcidic:', isAcidic);
+console.log('type of isAcidic:', typeof isAcidic);
+const numberOfSoldiers = 14;
+numberOfSoldiers === 300;
```

Comment:
this should be `const isSparta =  numberOfSoldiers === 300` 
and then you log the isSparta variable.